webpack.config.js

```diff
@@ -0,0 +1,12 @@
+const pkg = require('./package.json');
+
+module.exports = {
+    entry: {
+        'injector.lite': './src/entries/injector.lite.js',
```

Comment:
I tried bundling the injector.lite.js file and there was an error. I was able to fix the error by updating the string-replace loader configuration in the grunt file.  It is currently configured to replace window.FAI with CNN in all files named injector.js.  It needs to be updated to replace window.FAI with CNN in all files named injector.lite.js.